stimuli-ci-generation/packages

```diff
@@ -0,0 +1 @@
+matlab
```

Comment:
The comments say this file will get appended with packages from Kaggle? Is this still true? Why do we need to go to Kaggle everytime to fetch packages?

Reply:
We DON'T want to do that-- that was one of the problems.